packages/@aws-cdk/aws-eks/lib/kubectl-layer.ts

```diff
@@ -1,90 +1,15 @@
-import * as crypto from 'crypto';
```

Comment:
Need to remove documentation about this from the [README](https://github.com/aws/aws-cdk/tree/master/packages/%40aws-cdk/aws-eks#runtime).

Also - are we intentionally not allowing for custom lambda layers? We could possible allow the user to provider the own [`lambda.Code`](https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/aws-lambda/lib/code.ts#L11) construct.

Reply:
Yes, this is an implementation detail of our custom resources, which are an implementation detail of the EKS module. Users should technically not care about what's installed in there and I rather we don't expose those degrees of freedom unless they are actually required.

We can always add that at a later stage.

Reply:
Updated README